### PR TITLE
add script to convert nebula catalog text

### PR DESCRIPTION
**Pull request by silas1037**
It is an easy tool rewrited under python scripts, from [My Checking Program](https://github.com/silas1037/Stellarium-nebula).

### CODE REVIEW

**Comment by alex-w** [comment]
Stellarium has itself code for converting the catalog, so, is this script really need?

**Comment by silas1037** [comment]
Well, it provides both encoding and decoding functions. Maybe some users would like to decode catalog such as extended one, or want to do to converting test without waiting the long starting up.

**Comment by alex-w** [comment]
See also: https://github.com/Stellarium/stellarium/discussions/2317

File: nebula-catalog/catalog-converter.py [hunk 1]
```diff
@@ -0,0 +1,452 @@
...
+            dsoOutStream.writeQString(vp.group(2))
+            addedHead=True
+
+        if (record.startswith("//") or record.startswith("#")):
+            totalRecords-=1
+            continue
+        
+        # write when could not get label from txt
+        if not addedHead:
+            dsoOutStream.writeQString('3.13')
```
Comment (alex-w) [inline nebula-catalog/catalog-converter.py]: Why the version is hard coded?

File: nebula-catalog/catalog-converter.py [hunk 2]
```diff
@@ -0,0 +1,452 @@
...
+            dsoOutStream.writeQString('standard')
```
Comment (alex-w) [inline nebula-catalog/catalog-converter.py]: Well, if catalog are modified then he cannot be marked as 'standard'

**Comment by silas1037** [comment]
Alright, I change it to parameter, because they only effect when text has no head note.

**Review by alex-w — APPROVED** [review]

**Pull request merged**